react/AsidedLayout/AsidedLayout.js

```diff
@@ -3,13 +3,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
+const LEFT = 'left';
+const RIGHT = 'right';
+
 const defaultRenderAside = () => null;
 
-const conditionallyRenderAside = (condition, renderAside, classNameAside, size) => (
+const conditionallyRenderAside = (condition, renderAside, classNameAside, size, position) => (
   condition ?
     <div
       className={classnames({
         [classNameAside]: classNameAside,
+        [styles.leftAside]: position === LEFT,
```

Comment:
could just userd poisition here like [styles[`${position}Aside`]]

Reply:
I probably prefer the implemented solution as it's lint-able and correlates with the dom position behavior when it errors:
e.g. if the prop `lefft` was used React will show a console error but still render, and rendered output would be the same as if the prop was undefined.